### DISCUSSION TITLE
Is there a way not to trigger the dag when I change the airflow dags to `paused -> unpaused`?

### DISCUSSION THREAD

**Discussion by jojaeng2** [apache/airflow#37957] [Q&A]
Hello. First of all, I understand the concept of start_date, interval in airflow. 

The usability I want is as follows

1. Set the start_date to 1900,1,1.
The reason for this is that it was rather confusing to specify different start_date for each dag.

2. The interval is taken differently for each dag.

3. Dag does not run when I change the tag from a paced state to an unpaired state.

Number 3 is the problem I want to solve. 
Even though catchup is false, the moment i change it to unpaused, the tag triggers immediately. 
I understood that when it is unpaused, it works because believe that the current dag should be run based on the last time the dag was triggered.

Is there any way I can make it work the way I want it to?
Thank you.

**Comment by jojaeng2** [comment]
Similar issue : [issue](https://github.com/apache/airflow/issues/24534)

**Comment by tanelk** [comment]
`CronTriggerTimetable` with `catchup=False` should be what you are looking for:
https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html#differences-between-the-two-cron-timetables

**Reply by jojaeng2** [reply]
Thank you for your advice. I tried the way you told me about catchup false, but it doesn't work. For the first dag I created, it still runs once when I change it to enable..

**Comment by jojaeng2** [comment]
![image](https://github.com/apache/airflow/assets/76645095/ef6451a0-ed3a-49f5-ad09-0df4cebfd763)
I think the problem is that the next_dag_run of the first dag I created is looking at yesterday. Hmm.. I think I need to add some settings, but I don't get the hang of it
